tidb-cloud/manage-user-access.md

```diff
@@ -1,135 +1,159 @@
 ---
-title: Manage Console User Access
-summary: Learn how to manage the user access of the TiDB Cloud console.
+title: Identity Access Management
+summary: Learn how to manage identity access in TiDB Cloud.
 ---
 
-# Manage Console User Access
+# Identity Access Management
 
-This document describes how to manage user access of the [TiDB Cloud console](https://tidbcloud.com/console).
+This document describes how to manage access to organizations, projects, roles, and user profiles in TiDB Cloud.
 
-## Sign in
+Before accessing TiDB cloud, [create a TiDB cloud account](https://tidbcloud.com/free-trial). You can either sign up with email and password so that you can [manage your password using TiDB Cloud](/tidb-cloud/tidb-cloud-security-password-authentication.md), or choose your Google Workspace or GitHub account for single sign-on (SSO) to TiDB Cloud.
```

Comment:
```suggestion
Before accessing TiDB cloud, [create a TiDB cloud account](https://tidbcloud.com/free-trial). You can either sign up with email and password so that you can [manage your password using TiDB Cloud](/tidb-cloud/tidb-cloud-password-authentication.md), or choose your Google Workspace or GitHub account for single sign-on (SSO) to TiDB Cloud.
```